pkg/operators/oci-handler/oci.go

```diff
@@ -43,21 +42,20 @@ const (
 	publicKey             = "public-key"
 )
 
-type ociHandler struct{}
+type ociHandler struct {
+	globalParams *params.Params
+}
 
 func (o *ociHandler) Name() string {
 	return "oci"
 }
 
-func (o *ociHandler) Init(*params.Params) error {
+func (o *ociHandler) Init(params *params.Params) error {
+	o.globalParams = params
 	return nil
 }
 
 func (o *ociHandler) GlobalParams() api.Params {
-	return nil
-}
-
-func (o *ociHandler) InstanceParams() api.Params {
 	return api.Params{
 		// Hardcoded for now
```

Comment:
Not true anymore?

Reply:
I don't know what it addressed originally 😄 But I guess it can be removed.